cmd/nerdctl/restart.go

```diff
@@ -66,6 +73,12 @@ func restartAction(cmd *cobra.Command, args []string) error {
 			if err := stopContainer(ctx, found.Container, timeout); err != nil {
 				return err
 			}
+
+			if ls, err := found.Container.Labels(ctx); err == nil && strings.Contains(ls[labels.Networks], "container") {
+				if err := reconfigureNetContainer(ctx, found.Container, client); err != nil {
+					return err
+				}
+			}
```

Comment:
Move to `startContainer`, thanks

Reply:
I think If fits better in  between `stop` and `start` here?  `startContainer` is also called when `create` + `start`, which already specified network namespace when the `create` command calls `createContainer`

Reply:
How do?
- nerdctl run -d --name a
- nerdctl run -d --net container:a --name b
- nerdctl restart a
- nerdctl stop b
- nerdctl start b

Reply:
good catch!  will also add corresponding tests